src/lualib/ArrayConcat.ts

```diff
@@ -1,4 +1,4 @@
-function __TS__ArrayConcat(this: void, arr1: any[], ...args: any[]): any[] {
+export function __TS__ArrayConcat(this: void, arr1: any[], ...args: any[]): any[] {
```

Comment:
Is it possible to make Array a class like set/map as well?
To give people the ability to overwrite array built ins.

EDIT: same applies to Object

Reply:
that might cause problems when interacting with existing lua code. Could be done in separate PR, see #262

Reply:
i think it would be annoying for end users to have to mix Array and LuaArray, for example my game API returns lots of LuaArray